### ISSUE TITLE
Exemple en cours ne compile pour architecture ESP32

### ISSUE THREAD

**Issue by yheynema** [TechnoPhysCAL/TGP_Ecran#2]
Lorsque l'on cut&paste l'exemple à partir du github (raw copy de ecran_simple.ino) et que l'on compile dans l'environnement 2.3.2 avec la version 1.3.0 de la librairie, nous obtenons une erreur de compilation pour un l'arch ESP32 (DOIT).

Le cas semble etre relié entre la version 1.3.0 qui est installé par l'IDE Arduino et l'exemple de code le plus récent "commit" sur github. 

C:\Users\08041\AppData\Local\Temp\.arduinoIDE-unsaved2024413-36652-vdhyl9.mg57p\sketch_may13a\sketch_may13a.ino:16:48: error: no matching function for call to 'Ecran::Ecran(int, TwoWire*, int)'
 Ecran monEcran(OLED_RESET,&Wire,DEFAULT_ADDRESS); //Création de l'objet 'ecran' de classe Ecran

Faudrait p-ê que les développement se fasse sur une autre branche que la master?

**Comment by dualB** [comment]
Effectivement, j'ai poussé des modifs sur la branche master, sans m'en rendre compte.
En théorie, les étudiants utilisent la version installée par l'IDE, et non par clonage.
Je vais transférer sur une autre branche.
